runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1462,13 +1462,10 @@ NNFW_STATUS nnfw_session::train_export_circle(const char *path)
     bool ensure_mmap()
     {
       struct stat file_stat;
-      if (fstat(_fd, &file_stat) != 0)
+      if (fstat(_fd, &file_stat) != 0 || file_stat.st_size < 0 ||
+          static_cast<size_t>(file_stat.st_size) > SIZE_MAX)
```

Comment:
Is always `static_cast<size_t>(file_stat.st_size) > SIZE_MAX` true?

Reply:
:cry: You're right. `size_t` should be `uint64_t`.